community/mtd-utils/APKBUILD

```diff
@@ -0,0 +1,40 @@
+# Maintainer: Olliver Schinagl <oliver@schinagl.nl>
+pkgname=mtd-utils
+pkgver=2.1.0
+pkgrel=0
+pkgdesc="Utilities for handling MTD devices, and for dealing with FTL, NFTL JFFS2, etc."
+url="http://www.linux-mtd.infradead.org/"
+arch="all"
+license="GPL-2.0"
```

Comment:
```suggestion
license="GPL-2.0-or-later"
```

Reply:
can't change the license; the license been the same since 1991 and it's not or later from what I can see

Reply:
Only the following SPDX identifiers are acceptable for a GNU-GPL license:
GPL-3.0-or-later
GPL-3.0-only
GPL-2.0-or-later
GPL-2.0-only
GPL-1.0-or-later
GPL-1.0-only

GPL-2.0, is not. I looked at mtd-utils' COPYING file and it is the GPL-2.0-or-later variant.

Reply:
See the [spdx licence list][0] for reference

[0]:https://spdx.org/licenses/